app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            spouse = Person.people.get(person["wife"])
+            if spouse:
+                current_person.wife = spouse
```

Comment:
The code assumes that `current_person` has a `wife` attribute, but this attribute is not initialized in the `Person` class. Consider initializing `wife` and `husband` attributes in the `Person` class constructor.